scss/_base_forms-range.scss

```
@@ -37,7 +37,7 @@
       border-radius: $thumb-radius;
       box-shadow: $thumb-shadow;
       height: $thumb-size;
-      margin-top: ((-$thumb-size + $track-height) / 2) - $track-border-size;
+      margin-top: math.div((-$thumb-size + $track-height), 2) - $track-border-size;
```

In cases like these when the math is simple I'd change it to `* 0.5` to avoid function calls.

```suggestion
      margin-top: ((-$thumb-size + $track-height) * 0.5) - $track-border-size;
```